src/views/Pages/Register/Register.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { Button, Card, CardBody, Col, Container, Form, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+import { Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import { connect } from 'react-redux'
+import { compose } from 'redux'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['redux' should be listed in the project's dependencies. Run 'npm i -S redux' to add it](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)